docs/06_how-to-guides/30_key-value-api/kv_table/20_how-to-create-indexes-kv-table.md

```diff
@@ -54,14 +54,14 @@ class [[eosio::contract]] smrtcontract : public contract {
 
 ## Procedure
 
-### Define a unique index on property account_name using the macro KV_NAMED_INDEX
+### Define a unique index on the data member account_name using the macro KV_NAMED_INDEX
 
 1. Use the `KV_NAMED_INDEX` macro with two parameters.
-2. Pass the name of the index as the first parameter. The parameter must be a qualified `eosio::name`. See documentation for the [eosio name restrictions](https://developers.eos.io/welcome/latest/glossary/index#account-name).
-3. Pass the name of the property for which the index is defined as the second parameter.
-4. Call `init()` of the base class in the constructor of `address_table` type and pass the contract name as the first parameter and the primary index defined previously as the second parameter.
+2. The first parameter is the name of the index and must be a qualified `eosio::name`. See documentation for the [eosio name restrictions](https://developers.eos.io/welcome/latest/glossary/index#account-name).
+3. The second parameter is the name of the data member the index is defined for.
+4. Call the base class method `init()` in the constructor of `address_table` type and pass the contract name as the first parameter and the `account_name` index, using the KV_NAMED_INDEX macro, as the second parameter.
```

Comment:
This is not clear 

Call the base class method `init()` in the constructor of `address_table`. Pass the contract name as the first parameter. Pass `account_name`, matching the  KV_NAMED_INDEX macro name defined above, as the second parameter.

Reply:
In the constructor for the address_table class call the base class method contract::init, passing in the contract name (-> where does this come from / is it arbitrary?) and the index name. 

Explain which variable name is the index name when using the macro -> the description doesn't match! You say the  name of the index is the first parameter = "accname" but pass in account_name - the second parameter. It is very unclear. This is true for all the 4 options.

Reply:
you have a person struct and an address_table class -> this does not make sense

Reply:
yes not clear, I re-worked it.